### PR TITLE
mix function and fat arrow syntax

### PR DESCRIPTION
**Pull request by michaelchiche**

### CODE REVIEW

File: _posts/2016-07-27-custom-form-controls-in-angular-2.md [hunk 1]
```diff
@@ -558,7 +558,7 @@ Next, we somehow have to pass these values to our `validateCounterRange(c: FormC
 {% highlight js %}
 {% raw %}
 export function createCounterRangeValidator(maxValue, minValue) {
-  return function validateCounterRange(c: FormControl) => {
+  return function validateCounterRange(c: FormControl) {
```
Comment (ThomasBurleson) [inline _posts/2016-07-27-custom-form-controls-in-angular-2.md]: @michaelchiche  how about:

```js
return validateCounterRange(c: FormControl) => {
  ...
}
```

Comment (michaelchiche) [inline _posts/2016-07-27-custom-form-controls-in-angular-2.md]: its the same thing, i chose to stick with the function syntax and not the fat arrow one.

Comment (ThomasBurleson) [inline _posts/2016-07-27-custom-form-controls-in-angular-2.md]: I understand... my comment was suggesting that a closure is best represented (in ng2 codebases) with a fat-arrow syntax; as is seems (IMO) more clean.

Comment (michaelchiche) [inline _posts/2016-07-27-custom-form-controls-in-angular-2.md]: OK, but IMO keeping the function's name helps better follow the modifications for a beginner/new to angular2 person what is changed through the tutorial

**Pull request merged**